lib/committee/rails/test/methods.rb

```diff
@@ -4,13 +4,20 @@ module Methods
       include Committee::Test::Methods
 
       def committee_schema
+        @committee_schema = committee_options[:schema]
```

Comment:
Should I also set nil guard here ? like `@committee_schema ||= ...`

Reply:
I think you don't have to  add nil guard. `committee_options[:schema]` seems not to cost.